### PR TITLE
Starting New features for Telco Core 413 RN

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1123]: Document new features

Version(s):
4.13 

Issue:
https://issues.redhat.com/browse/TELCODOCS-1123

Link to docs preview:

- https://57258--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-supporting-ocp-nodes-with-dual-port-nics
- https://57258--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-configuring-power-states-using-ztp

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -257,14 +268,14 @@ For more information, see xref:../machine_management/control_plane_machine_manag
...
-You can now use {op-system-first} image layering to add {op-system-base-full} and third-party packages to cluster nodes. 
+You can now use {op-system-first} image layering to add {op-system-base-full} and third-party packages to cluster nodes.
 
 For more information, see  xref:../post_installation_configuration/coreos-layering.adoc#coreos-layering[{op-system-first} image layering].
 
 [id="ocp-4-13-machine-config-operator-core"]
 ==== Support for setting the `core` user password
 
-You can now create a password for the {op-system} `core` user. In the event that you cannot use SSH or the `oc debug node` command to access a node, this password allows you to use the `core` user to access the node through a cloud provider serial console or a bare metal baseboard controller manager (BMC). 
+You can now create a password for the {op-system} `core` user. In the event that you cannot use SSH or the `oc debug node` command to access a node, this password allows you to use the `core` user to access the node through a cloud provider serial console or a bare metal baseboard controller manager (BMC).
```
Comment (tmalove) [inline release_notes/ocp-4-13-release-notes.adoc:278]:
```suggestion
You can now create a password for the {op-system} `core` user. If you cannot use SSH or the `oc debug node` command to access a node, this password allows you to use the `core` user to access the node through a cloud provider serial console or a bare metal baseboard controller manager (BMC).
```
See the IBMSG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-13-release-notes.adoc (hunk 1 above)
Comment (kquinn1204) [reply release_notes/ocp-4-13-release-notes.adoc:278]: I will update but to be honest this is not content I created.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -370,6 +381,15 @@ This update adds the following features:
 
 For more information, see xref:../scalability_and_performance/cnf-numa-aware-scheduling.adoc#cnf-about-numa-aware-scheduling_numa-aware[Scheduling NUMA-aware workloads].
 
+[id="ocp-4-13-configuring-power-states-using-ztp"]
+==== Configuring power states using zero touch provisioning (ZTP)
+
+{product-title} {product-version} introduces the ability to configure using zero touch provisioning (ZTP) the various power state configurations for critical and non-critical workloads.
```
Comment (sakhoury) [inline release_notes/ocp-4-13-release-notes.adoc:387]: Please reword this to indicate that OCP 4.12 had introduced the ability to set power states for critical and non-critical workloads. This release now enables the user to configure power states using ZTP.

**Review by sakhoury — CHANGES_REQUESTED** [review]
Reviewing `Configuring power states using zero touch provisioning (ZTP)` only.

File: release_notes/ocp-4-13-release-notes.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (kquinn1204) [reply release_notes/ocp-4-13-release-notes.adoc:387]: Resolved by comment below

**Comment by kquinn1204** [comment]
@mjpytlak This can hopefully be merged now

**Comment by mjpytlak** [comment]
@kquinn1204 I believe we are still waiting on @sakhoury approval.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -370,6 +381,13 @@ This update adds the following features:
 
 For more information, see xref:../scalability_and_performance/cnf-numa-aware-scheduling.adoc#cnf-about-numa-aware-scheduling_numa-aware[Scheduling NUMA-aware workloads].
 
+[id="ocp-4-13-configuring-power-states-using-ztp"]
+==== Configuring power states using zero touch provisioning (ZTP)
+
+{product-title} 4.12 introduced the ability to set power states for critical and non-critical workloads. This release now enables the user to configure power states by using ZTP.
+
+For more information about the feature, see xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.html#ztp-using-pgt-to-configure-power-saving-states_ztp-advanced-policy-config[Configuring power states using PolicyGenTemplates CRs].
```
Comment (mjpytlak) [inline release_notes/ocp-4-13-release-notes.adoc:389]:
```suggestion
For more information about the feature, see xref:../scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc#ztp-using-pgt-to-configure-power-saving-states_ztp-advanced-policy-config[Configuring power states using PolicyGenTemplates CRs].
```

**Comment by kquinn1204** [comment]
@mjpytlak Thank you got LGTM and fixed that mistake so should be OK fingers crossed

File: release_notes/ocp-4-13-release-notes.adoc [hunk 4]
```diff
@@ -244,6 +244,17 @@ With this update, you can assign IP addresses from a MetalLB `IPAddressPool` res
...
+[id="ocp-4-13-supporting-ocp-nodes-with-dual-port-nics"]
+==== Supporting {product-title} installation on nodes with dual-port NICs (Technology Preview)
+
+With this update, {product-title} cluster can be deployed on a bond interface with 2 virtual function (VFs) on 2 physical functions (PFs) using the following methods:
+
+* Agent-based installer
+* Installer-provisioned infrastructure installation
+* User-provisioned infrastructure installation
+
+For more information about installing {product-title} on nodes with dual-port NICs, see xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.html#nw-sriov-dual-nic-con_preparing-to-install-on-bare-metal[NIC partitioning for SR-IOV devices].
```
Comment (mjpytlak) [inline release_notes/ocp-4-13-release-notes.adoc:256]:
```suggestion
For more information about installing {product-title} on nodes with dual-port NICs, see xref:../installing/installing_bare_metal/preparing-to-install-on-bare-metal.adoc#nw-sriov-dual-nic-con_preparing-to-install-on-bare-metal[NIC partitioning for SR-IOV devices].
```
@kquinn1204 Sorry man. This one got by me the last time.

Comment (kquinn1204) [reply release_notes/ocp-4-13-release-notes.adoc]: Sorry my bad I should have noticed that @mjpytlak hopefully ok now
